packages/solid/.storybook/preview.tsx

```diff
@@ -57,10 +58,16 @@ const preview = {
     }
   },
   decorators: [
-    Story => {
+    (Story, context) => {
       const solidRoot = document.createElement('div');
+      if (shouldReload) {
+        shouldReload = false;
+        window.location.reload();
+      }
+
+      shouldReload = true;
       // teardown previous render (cleans up keyhandling)
-      dispose && dispose();
+      //dispose && dispose();
```

Comment:
should this still be commented out?

Reply:
Yea - it isn't needed now that the page will reload.